src/main/resources/swagger/api-docs.yaml

```diff
@@ -75,18 +75,24 @@ paths:
       tags:
         - Admin
 
-  /api/google/user/petServiceAccount:
+  /api/google/user/petServiceAccount/{project}:
```

Comment:
does this mean the old API is gone? if so, rolling out this version of sam will break leo...

Reply:
Old api is still there just undocumented. It doesn't have google in the uri.